src/site/_utils/get-image-path.js

```diff
@@ -26,7 +26,9 @@ const site = require("../_data/site");
  * it's a production environment.
  */
 module.exports = function getImagePath(src, pageUrl) {
-  let imagePath = path.join(pageUrl, src);
+  let imagePath = (src || "").startsWith("~/")
+    ? src.replace("~/", "/")
```

Comment:
```
src.replace(/^~\//, "/")
```

to make sure we only catch the left.

By the way, if we're just replacing the "root" with "/" anyway—why do we need the tilde?

Reply:
The tilde is just used to make the distinction it should be from root. For posts the photo is relative to the folder the photo is in, so this function joins the directory to the image src. Adding the `~` just says not to do that. Otherwise I'll make the regex change.

Reply:
I don't want to boil the ocean, but it feels like we're working around the fact that the PostCard and PathCard elements aren't very extensible. I see this happening on the podcast page as well (https://github.com/GoogleChrome/web.dev/pull/2368#issuecomment-601855753).

I'm concerned about us shipping workarounds that create technical debt in the long run. I keep wondering if we should instead look at writing an [eleventy transform](https://www.11ty.dev/docs/config/#transforms) that converts _all_ image paths to be from the CDN if it's in the prod environment and hopefully our components wouldn't need to manually call and request CDN paths.

Also I noticed the images that are being served on the authors page seem to have an intrinsic size of 192 x 192 which seems quite small. Basically they all look pixelated and I'm assuming that's because we never asked anyone to upload an image in these larger dimensions. So I'm kind of wondering if we want to go with this design at all. We could omit the hero image and go with their name and try and work in the smaller profile image..? Or we'll want to ask everyone to upload a higher res headshot.